SPECS/dnf5/dnf5.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Don't forget to update the License Map file.  I believe this SPEC file came from the dnf5 repo directly?

Reply:
Yes, it came from the repo directly. Who would the attribution be to rpm-software-management, dnf5 source, or other?

Reply:
"DNF5 Authors" is my pick, but whatever works!

Reply:
Landed on "RPM software management source" after talking with Pawel similar to sysbench